File: solutions/python3/easy/py-introduction-to-sets/solution-approach1.py
# ──────────────────────────────────────────────────
# Problem     Introduction to Sets
# Difficulty  Easy
# Subdomain   Sets
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-06-24, 03:05 p.m.
# ──────────────────────────────────────────────────

def average(array):
    # your code goes here
    unique = list(set(array))
    return sum(unique)/len(unique)

File: solutions/cpp/easy/c-tutorial-for-loop/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/c-tutorial-for-loop/problem?isFullScreen=true
// Problem     For Loop
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-07-24, 10:46 a.m.
// ──────────────────────────────────────────────────

#include <iostream>
#include <cstdio>
using namespace std;
string numbers[] = {"", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
int main() {
    // Complete the code.
    int a,b;
    cin >> a >> b;
    
    for(int i=a; i <=b; i++)
    {if(i>=1 && i<= 9){
        cout << numbers[i] << endl;
    }else{
        if(i % 2 == 0){
            cout<<"even"<<endl;
        }else{cout << "odd" << endl;
        }
    }
    }
    return 0;

}

